zebedee-cms/src/main/java/com/github/onsdigital/zebedee/permissions/service/JWTPermissionsServiceImpl.java

```diff
@@ -0,0 +1,391 @@
+    @Override
+    public boolean canView(User user, CollectionDescription collectionDescription) throws
+            UnsupportedOperationExceptions {
+        throw new UnsupportedOperationExceptions("JWT Permissions service error for canView no longer required");
+    }
+
+    /**
+     * @param email                 the email of the user to check.
+     * @param collectionDescription the {@link CollectionDescription} of the {@link Collection} to check.
+     * @return null
+     * @throws UnsupportedOperationExceptions will error if invoked.
+     */
+    @Override
+    public boolean canView(String email, CollectionDescription collectionDescription) throws
+            UnsupportedOperationExceptions {
+        throw new UnsupportedOperationExceptions("JWT Permissions service error for canView no longer required");
+    }
+
+    /**
+     * @param collectionDescription the {@link CollectionDescription} of the {@link Collection} in question.
+     * @param teamId                the {@link Team} to permit view permission to.
+     * @param session               the {@link Session} of the user granting the permission.
+     * @throws UnsupportedOperationExceptions will error if invoked.
+     */
+    @Override
+    public void addViewerTeam(CollectionDescription collectionDescription, Integer teamId, Session session) throws
+            UnsupportedOperationExceptions {
+        throw new UnsupportedOperationExceptions("JWT Permissions service error for addViewerTeam no longer required");
+    }
+
+    /**
+     * this method is being migrated to the dp-identity-api
+     * session and collection validation is in canview module
+     *
+     * @param collectionDescription the {@link CollectionDescription} of the {@link Collection} to get the viewer
+     *                              teams for.
+     * @param session               the {@link Session} of the {@link User} requesting this information.
+     * @return null
+     * @throws IOException, ZebedeeException { will error if invoked.
+     */
+    @Override
+    public Set<Integer> listViewerTeams(CollectionDescription collectionDescription, Session session) throws
+            IOException, UnauthorizedException {
+        if (!canView(session, collectionDescription)) {
+            throw new IOException("JWT Permissions service error for listViewerTeams input error ");
+        }
+
+        List<Integer> sessionsTeamsList = convertGroupsToTeams(session);
+        List<Integer> collectionTeamIntegerList = convertTeamsListStringToListInteger(collectionDescription);
+
+        Set<Integer> resultTeamIds = new HashSet<>();
+        for (Integer sessionTeamID : sessionsTeamsList) {
+            for (Integer collectionTeamIds : collectionTeamIntegerList) {
+                if (sessionTeamID.equals(collectionTeamIds)) {
+                    resultTeamIds.add(sessionTeamID);
+                }
+            }
+        }
+
+        return java.util.Collections.unmodifiableSet(resultTeamIds);
```

Comment:
Im not 100% sure this logic is right for what we need to do? I think this needs to return a list of team IDs for the teams that have access to the collection. If that is correct then we don't need the session as this will be the user's permissions.